src/main/java/com/github/davidmoten/rtree/RTree.java

```diff
@@ -437,9 +407,8 @@ private double mid(HasGeometry o) {
 
     /**
      * Returns an immutable copy of the RTree with the addition of given entry.
-     * 
-     * @param entry
-     *            item to add to the R-tree.
+     *
```

Comment:
please don't reformat javadoc, it's got nothing to do with your PR

Reply:
OK. In case of the static methods named "of" personally I prefer fully name, but in this case only Optional.of is used.
I will also check javadoc against master.